tdrs-backend/Pipfile

```diff
@@ -4,24 +4,24 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-awscli = "1.27.5"
-awscli-local = "*"
-factory-boy = "*"
-flake8 = "*"
-flake8-docstrings = "*"
-isort = "*"
-mkdocs = "*"
-pygraphviz = "*"
-pyparsing = "*"
-pytest = "*"
-pytest-cov = "*"
-pytest-django = "*"
-pytest-mock = "*"
-pytest-factoryboy = "*"
+awscli = "1.29.4"
+awscli-local = "0.20"
+factory-boy = "==3.2.1"
+flake8 = "6.0.0"
+flake8-docstrings = "1.7.0"
+isort = "5.12.0"
+mkdocs = "1.4.3"
+pygraphviz = "1.11"
+pyparsing = "3.1.0"
+pytest = "7.4.0"
+pytest-cov = "4.1.0"
+pytest-django = "4.5.2"
+pytest-mock = "==3.11.0"
+pytest-factoryboy = "2.5.1"
 
```

Comment:
Way to go to find the issue! Really proud of you @elipe17 ... 

I don't think this setup is a long term solution. It's is a ton of work to stay up to date with all of these packages and bump up to necessary security/functionality/quality-of-life versions as they come out. 

Hopefully the incompatible latest versions will sync up and be compatible again soon. I wonder if we can move the rest to allow for minor patch fuzzy matches `~ instead of =` and get the offending packages to the same system once they are no longer in contention.